clientLibrary/src/main/scala/za/co/absa/clientexample/Application.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.clientexample
+
+import za.co.absa.clientexample.config.ConfigProvider
+import za.co.absa.loginclient.authorization.JWTDecoderProvider
+import za.co.absa.loginclient.tokenRetrieval.service.RetrieveToken
+
+import java.util.Scanner
+
+object Application {
```

Comment:
I am a bit puzzled about what is the purpose of this application class, and I see two possibilities:

1. Demonstrate how to use the client library
2. Provide means of running the library directly (as a simple test) via shell instead of actually using as a dependent library programmatically

It is somewhat unusual to combine both. I think it would be better if we could separate the library module and ideally publish it, e.g. similar how https://github.com/AbsaOSS/cobrix/blob/master/publish.sbt does it. And then have another module with the example usage -- that module would show the module usage as external library or at least as an external module (depending on a just built snapshot would be fine).

Reply:
Done